ast/src/object/mod.rs

```diff
@@ -41,7 +41,7 @@ pub struct PILGraph {
     pub main: Machine,
     pub entry_points: Vec<Operation>,
     pub objects: BTreeMap<Location, Object>,
-    pub definitions: BTreeMap<AbsoluteSymbolPath, TypeOrExpression>,
+    pub definitions: BTreeMap<AbsoluteSymbolPath, Vec<PilStatement>>,
```

Comment:
Is the key here the module name?

Reply:
yes, comment added

Reply:
Maybe we should also rename this to `statements`?

Reply:
done